bin/build_image.sh

```diff
@@ -55,3 +55,4 @@ else
   qemu-img convert -o compat=0.10 -O qcow2 -c ${DIR_NAME}/${IMAGE_NAME} \
     $FINAL_QCOW_FILE_NAME
 fi
+
```

Comment:
random extra line?

Reply:
Will remove and commit with any other changes that you suggest. Don't want to trigger another build right now.